apps/srt-live-transmit.cpp

```diff
@@ -771,12 +777,34 @@ int main(int argc, char** argv)
                     break;
                 }
 
+
+                bool srcReady = false;
+
+                if (src.get() && src->IsOpen())
```

Comment:
Maybe this way then if you still want to use `cin.eof()`?
```suggestion
                if (src.get() && src->IsOpen() && !src->EOF())
```